docs/reference/filters.md

```diff
@@ -730,6 +730,27 @@ Example:
 * -> logHeader("request", "response") -> "https://www.example.org";
 ```
 
+### logBody
+
+The `logBody` filter logs the request or response body while
```

Comment:
Should we mention that it may log body in chunks, i.e. multiple log lines per requets/response?

Reply:
yes done